decidim-process-extended/decidim-process-extended.gemspec

```diff
@@ -16,6 +16,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*", "MIT-LICENSE", "Rakefile", "README.md"]
 
   s.add_dependency "rails", ">= 5.2", "< 6.0.x"
+  s.add_dependency "decidim-core"
```

Comment:
Why this new dependency?

Reply:
for some new functionalities in decidim-core